compute/functions/api-cli/fun-uploading-with-serverless-framework.mdx

```diff
@@ -225,3 +225,7 @@ This removes all your functions, as well as your Scaleway Functions' namespace.
   Your Scaleway Functions Registry namespace is not automatically removed. You must delete it manually either via the Scaleway console, or Scaleway's Functions Registry API.
  
 </Message>
```

Comment:
there seems to be an issue with the code here

Reply:
Yes I thought that, but in VSCode it looks fine, and the page looks/works fine with a yarn develop...